apps/site/pages/docs/migration-guides/Migrating to 2.2.65 - graphql-utils depreciation.mdx

```diff
@@ -0,0 +1,111 @@
+# Migrating to 2.2.65 - graphql-utils depreciation
+
+From 2.2.60 on, the `@faststore/graphql-utils` is officially deprecated. It was previously used internally by `@faststore/core` to handle GraphQL optimizations and GraphQL definitions.
+
+The `@faststore/graphql-utils` has been replaced by open-source solutions maintained by the community that are now re-exported from `@faststore/core`. There are minor breaking changes on how developers should write GraphQL definitions and invoke queries and mutations, which were introduced in version 2.2.60.
+
+## The `gql` helper
+
+Previously, it was possible to import the `gql` helper function directly from `@faststore/graphql-utils` - although it was not recommended. This will now throw an error. See the example below:
```

Comment:
Why was not recommended?

Reply:
Because users should import everything from @faststore/core, that's the centralized package that should include everything you need to build a store. Importing the same code from multiple sources may cause versioning errors when handling dependencies.